src/Model/Scope.php

```diff
@@ -188,6 +189,28 @@ public function toWords(Model $model = null): string
         return implode($glue, $parts);
     }
 
+    /**
+     * Creates the simplest scope object based on arguments.
+     *
+     * @param Scope\AbstractScope|string|array|\atk4\data\Field $field
+     * @param string                                            $operator
+     * @param mixed                                             $value
+     *
+     * @return Scope\AbstractScope
+     */
+    public static function create($field, $operator = null, $value = null)
```

Comment:
addCondition is more or less legacy design, do not introduce it back - user should use specific method!